components/uniset/src/provider.rs

```diff
@@ -14,78 +14,811 @@ pub mod key {
+    pub const CANONICAL_COMBINING_CLASS_BELOW_LEFT_V1: ResourceKey =
+        resource_key!(uniset, "2=218", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC22_V1: ResourceKey = resource_key!(uniset, "2=22", 1);
+    pub const CANONICAL_COMBINING_CLASS_BELOW_V1: ResourceKey = resource_key!(uniset, "2=220", 1);
+    pub const CANONICAL_COMBINING_CLASS_BELOW_RIGHT_V1: ResourceKey =
+        resource_key!(uniset, "2=222", 1);
+    pub const CANONICAL_COMBINING_CLASS_LEFT_V1: ResourceKey = resource_key!(uniset, "2=224", 1);
+    pub const CANONICAL_COMBINING_CLASS_RIGHT_V1: ResourceKey = resource_key!(uniset, "2=226", 1);
+    pub const CANONICAL_COMBINING_CLASS_ABOVE_LEFT_V1: ResourceKey =
+        resource_key!(uniset, "2=228", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC23_V1: ResourceKey = resource_key!(uniset, "2=23", 1);
+    pub const CANONICAL_COMBINING_CLASS_ABOVE_V1: ResourceKey = resource_key!(uniset, "2=230", 1);
+    pub const CANONICAL_COMBINING_CLASS_ABOVE_RIGHT_V1: ResourceKey =
+        resource_key!(uniset, "2=232", 1);
+    pub const CANONICAL_COMBINING_CLASS_DOUBLE_BELOW_V1: ResourceKey =
+        resource_key!(uniset, "2=233", 1);
+    pub const CANONICAL_COMBINING_CLASS_DOUBLE_ABOVE_V1: ResourceKey =
+        resource_key!(uniset, "2=234", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC24_V1: ResourceKey = resource_key!(uniset, "2=324", 1);
+    pub const CANONICAL_COMBINING_CLASS_IOTA_SUBSCRIPT_V1: ResourceKey =
+        resource_key!(uniset, "2=240", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC25_V1: ResourceKey = resource_key!(uniset, "2=25", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC26_V1: ResourceKey = resource_key!(uniset, "2=26", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC27_V1: ResourceKey = resource_key!(uniset, "2=27", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC28_V1: ResourceKey = resource_key!(uniset, "2=28", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC29_V1: ResourceKey = resource_key!(uniset, "2=29", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC30_V1: ResourceKey = resource_key!(uniset, "2=30", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC31_V1: ResourceKey = resource_key!(uniset, "2=31", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC32_V1: ResourceKey = resource_key!(uniset, "2=32", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC33_V1: ResourceKey = resource_key!(uniset, "2=33", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC34_V1: ResourceKey = resource_key!(uniset, "2=34", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC35_V1: ResourceKey = resource_key!(uniset, "2=35", 1);
+    pub const CANONICAL_COMBINING_CLASS_CCC36_V1: ResourceKey = resource_key!(uniset, "2=36", 1);
+    pub const CANONICAL_COMBINING_CLASS_HAN_READING_V1: ResourceKey =
+        resource_key!(uniset, "2=6", 1);
+    pub const CANONICAL_COMBINING_CLASS_NUKTA_V1: ResourceKey = resource_key!(uniset, "2=7", 1);
+    pub const CANONICAL_COMBINING_CLASS_KANA_VOICING_V1: ResourceKey =
```

Comment:
Optional: make the new rustfmt.toml file apply to this file, too

Reply:
Done. I reverted the change of `props.rs` -> `props/mod.rs` because `rustfmt.toml` wasn't being used by the Rust formatter, and because the Rust formatter config applies not only to `props.rs`, but also here to `provider.rs` and another file in the same directory.